canonical-web-and-design/snapcraft.io
# Remove unused js from brand store pages

## Done

Removed unused JS from brand store pages

## Issue / Card

Fixes #3278

## QA

- Pull the branch
- Run the site using the dotrun snap with `$ dotrun --env WEBAPP=limenet`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Check that the page works as expected
- Do a search and check that the page works as expected
